src/NLog/LayoutRenderers/BaseDirLayoutRenderer.cs

```diff
@@ -52,19 +52,22 @@ public class BaseDirLayoutRenderer : LayoutRenderer
         private readonly string _baseDir;
 
 #if !SILVERLIGHT && !NETSTANDARD1_3
-
         /// <summary>
         /// cached
         /// </summary>
         private string _processDir;
+#endif
 
         /// <summary>
         /// Use base dir of current process.
         /// </summary>
         /// <docgen category='Rendering Options' order='10' />
         public bool ProcessDir { get; set; }
 
-#endif
+        /// <summary>
+        /// Fallback to the base dir of current process, when AppDomain.BaseDirectory is Temp-Path (NetCore3 Single File Publish)
```

Comment:
Please use ".NET Core 3" - i'm trying hard to use one form, and this is the one MS advices

Reply:
Fixed